### PR TITLE
Adding Kill Tabs for Chromium, Chrome, Brave to Raycast

### PR DESCRIPTION
**Pull request by sereneinserenade**
## Description

Adding Kill Tabs for Chromium, Chrome, Brave to Raycast. It uses https://github.com/sindresorhus/kill-tabs under the hood for killing tabs.

## Screencast

- Tabs come back when reloaded.

https://user-images.githubusercontent.com/45892659/162641360-a35fb8d0-6299-46b8-a2ed-7c0d914a1c69.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/kill-tabs/package.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      "description": "Kill All Chrome Tabs",
+      "mode": "no-view"
+    },
+    {
+      "name": "killBraveTabs",
+      "title": "Kill Brave Tabs",
+      "subtitle": "Brave",
+      "description": "Kill All Brave Tabs",
+      "mode": "no-view"
+    }
```
Comment (thomaspaulmann) [inline extensions/kill-tabs/package.json:36]:
```suggestion
    {
      "name": "killChromiumTabs",
      "title": "Kill Tabs",
      "subtitle": "Chromium",
      "description": "Kill All Chromium Tabs",
      "mode": "no-view"
    },
    {
      "name": "killChromeTabs",
      "title": "Kill Tabs",
      "subtitle": "Chrome",
      "description": "Kill All Chrome Tabs",
      "mode": "no-view"
    },
    {
      "name": "killBraveTabs",
      "title": "Kill Tabs",
      "subtitle": "Brave",
      "description": "Kill All Brave Tabs",
      "mode": "no-view"
    }
```

**Review by thomaspaulmann — APPROVED** [review]
Thanks for the extension. I'm wondering if it would make sense to have a setting for the browser that you're using instead of having three separate commands. Thoughts?

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Can you please update your [screenshots](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) to follow our guidelines? Then they will show up in the store and it will make your extension look a lot better. Let me know what you think about the setting?

**Comment by pernielsentikaer** [comment]
Hi @sereneinserenade 👋 

Do you have time to finish this one? It also seems possible to kill the tabs in Edge; maybe include that?

Let us remove the metadata images in this case since it's not showing a command. Can you delete that?

**Comment by pernielsentikaer** [comment]
I'm closing this due to inactivity; feel free to open it again or comment in the thread when you're ready to continue working on it 🙂

You can also catch me here or in [Slack](https://www.raycast.com/community) if you want to discuss 😄

**Pull request closed without merging**